### PR TITLE
Feature/l33 t 794 enabling cfg config

### PR DESCRIPTION
**Pull request by notsaltydev**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -529,4 +530,53 @@ var config = {
     // no configuration value should follow this line.
 };
 
+if (isCfgEnabled) {
+    // Overwrite config here...
+}
+
+function parseURLParams(
```
Comment (notsaltydev) [inline config.js:537]: This is copy of: `react/features/base/config/parseURLParams.js`

File: .gitignore [hunk 2]
```diff
@@ -84,3 +84,4 @@ android/app/google-services.json
 ios/app/dropbox.key
 ios/app/GoogleService-Info.plist
 
+.tmp/
```
Comment (notsaltydev) [inline .gitignore:87]: remove this change.

File: config.js (hunk 1 above)
Comment (EnricoSchw) [inline config.js:551]: The url already should parsed if the config is loaded. In the 'config.js' only need to check a parameter

File: config.js [hunk 3]
```diff
@@ -529,4 +530,67 @@ var config = {
     // no configuration value should follow this line.
 };
 
+if (isCfgEnabled) {
+    // Overwrite config here...
+    config.p2p.stunServers = [
```
Comment (notsaltydev) [inline config.js:535]: https://github.com/jameda-telemedicine/jitsi-meet/blob/0f1dedd11c01540562c0ac7ffc63c6cca49274c7/config.js#L548

File: config.js (hunk 1 above)
Comment (notsaltydev) [reply config.js:551]: Do you mean to move this function outside this directory?

File: config_cfg.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-unused-vars, no-var */
+var isCfgEnabled = parseURLParams(window.location, true, 'search').cfg || false;
+
+function parseURLParams(
```
Comment (notsaltydev) [inline config_cfg.js:4]: I will change function implementation later.

**Pull request merged**